etc/insights-client.spec

```diff
@@ -157,10 +157,9 @@ fi
 mkdir -p /etc/redhat-access-insights
 ln -sf /etc/insights-client/insights-client.conf /etc/redhat-access-insights/redhat-access-insights.conf
 ln -sf /etc/insights-client/insights-client.cron /etc/redhat-access-insights/redhat-access-insights.cron
-ln -sf /etc/insights-client/.registered /etc/redhat-access-insights/.registered
-ln -sf /etc/insights-client/.unregistered /etc/redhat-access-insights/.unregistered
```

Comment:
What if the symlinks are already there from a previous installation?

Reply:
In that case should we delete these files on install? Shouldn't hurt since if they have an existing registration the API should echo back an OK and we'll regenerate them.